docs/Advanced-Usage/Generating-Different-Targets.rst

```diff
@@ -119,7 +119,7 @@ resides in ``sim/``.  These projects are:
    either BOOM or rocket pipelines, and should be your starting point if you're
    building an SoC with the Rocket Chip generator.
 2. **midasexamples**: the `Golden Gate example designs
-   <https://github.com/ucb-bar/midas-examples>`_, a set of simple chisel
+   <https://github.com/firesim/firesim/tree/dev/sim/src/main/scala/midasexamples>`_, a set of simple chisel
```

Comment:
Or, would you rather reference them like you do below without creating a hyperlink (see https://github.com/firesim/firesim/pull/717/files#diff-6ea46dd280d3c59fa262dd9fb27b7472124db0ceb33642c5e898d579ebda3707R152)